### PR TITLE
arakoon.1.8.6

### PR DESCRIPTION
**Pull request by toolslive**

### CODE REVIEW

File: packages/arakoon/arakoon.1.8.6/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+install: [make "install"   "OCAML_LIBDIR=%{arakoon:lib}%" "PREFIX=%{prefix}%"]
+
+remove:  [
+  [make "uninstall_client" "OCAML_LIBDIR=%{arakoon:lib}%" "PREFIX=%{prefix}%"]
+  ["rm" "-rf" "%{bin}%/arakoon"]
+]
+
+depends: [
+  "ocamlfind"      {build}
+  "bisect"         { >= "1.3"}
```
Comment (whitequark) [inline packages/arakoon/arakoon.1.8.6/opam]: Isn't this a `test` dependency?

File: packages/arakoon/arakoon.1.8.6/opam [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  "camlbz2"        { >= "0.6.0" }
+  "conf-libev"     { >= "4-11"}
+  "cstruct"        { >= "1.6.0"}
+  "ctypes"         { >= "0.4.0" }
+  "ctypes-foreign" { >= "0.4.0"}
+  "lwt"            { = "2.4.8" }  # 2.4.9 had an incompatible API change
+  "ocplib-endian"  { >= "0.8" }
+  "ssl"            { >= "0.5.0" }
+  "snappy"         { >= "0.1.0" }
+  "quickcheck"     { >= "1.0.2" }
```
Comment (whitequark) [inline packages/arakoon/arakoon.1.8.6/opam]: Ditto

File: packages/arakoon/arakoon.1.8.6/opam (hunk 1 above)
Comment (toolslive) [inline packages/arakoon/arakoon.1.8.6/opam]: yes. I'll update.

File: packages/arakoon/arakoon.1.8.6/opam (hunk 2 above)
Comment (toolslive) [inline packages/arakoon/arakoon.1.8.6/opam]: no. Arakoon just generates 1 binary. You can run that binary 
- as a node of an arakoon cluster
- for tests
- as an ad-hoc cli client
- as an administrative tool
- ...

**Pull request merged**